### ISSUE TITLE
Empty feeds with no error message

### ISSUE THREAD

**Issue by nooraibk** [j-fbriere/squawker#186]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/j-fbriere/squawker/releases).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/j-fbriere/squawker/issues) or [closed](https://github.com/j-fbriere/squawker/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug, that only occurs in Squawker

### Affected version

v3.6.4, v3.6.5

### Steps to reproduce the bug

1. Open App

### Expected behavior

Show feeds.

### Actual behavior

I'm getting empty fragment with no error message or feeds. v3.6.4 was working fine until one day it wasn't. Just updated to v3.6.5 but same bug.

**Comment by zer0fl0w** [comment]
#178 + #179

**Comment by j-fbriere** [comment]
The reason this error appears is because it is not possible to create guest accounts on Twitter/X anymore.
The version [v3.7.0](https://github.com/j-fbriere/squawker/releases/tag/v3.7.0) allows you to use a regular Twitter/X account.
To add the information (username/password) of your regular Twitter/X account, go to Settings / Account and tap on the plus icon. 
Since each account is rate limited (same with the guest accounts), you can add as many as you want.
This [wiki page](https://github.com/j-fbriere/squawker/wiki/3.-Regular-Twitter-X-accounts) explains how to create a Twitter/X account that is privacy oriented.

**Comment by hamza-abitaleb** [comment]
Also after using a regular account, i have this issue, not just at the previous version but also at the current 3.7.1 version

**Comment by j-fbriere** [comment]
@hamza-abitaleb Your feed do not work. 
Is it only the feed that do not work? 
Do the search user works? 
Do the subscription profile work? 
Thanks for the info.

**Comment by hamza-abitaleb** [comment]
> @hamza-abitaleb Your feed do not work. Is it only the feed that do not work? Do the search user works? Do the subscription profile work? Thanks for the info.

First i would like to thank you very much for this huge effort while i didn't regard also didn't thank you as should be and as i have to and as it must be till now, for all this effort

Second, answering questions:
No , i can't search for users also can't search for tweets by text , without any error , just no results message
But i can subscribe to profiles , as i have already a list of profiles and can see followers and find new profiles and subscribe to them without searching , 

And thank you very much again
And I hope that i have an ability to real support or help

**Comment by hamza-abitaleb** [comment]
Still having this issue with 3.7.2 version, also there is a notice, some options in settings make the application hang and stop working and get into no responding state if i active them while there is a connection to the internet, but if i am offline with those options , no hang state happen , and if i disabled those options the application work with no hang state also but with this empty feeds issue

**Comment by hamza-abitaleb** [comment]
> Still having this issue with 3.7.2 version, also there is a notice, some options in settings make the application hang and stop working and get into no responding state if i active them while there is a connection to the internet, but if i am offline with those options , no hang state happen , and if i disabled those options the application work with no hang state also but with this empty feeds issue even i am connected to the internet

They're the enhancements of X API options , 

i tried them and tried other options to determine which options exactly

**Comment by j-fbriere** [comment]
- Which Squawker version?
- Which options of X API are enabled (under Settings / General)?
- Do you have set a Twitters/X account (under Settings / Account)?

**Comment by BaconCondensate** [comment]
Running latest squawker (3.7.8), also encountering the issue of feed not loading.  Single account also works fine

**Comment by j-fbriere** [comment]
@BaconCondensate Are subscriptions working (going to Subscriptions page and selecting one subscription)?

**Comment by BaconCondensate** [comment]
> @BaconCondensate Are subscriptions working (going to Subscriptions page and selecting one subscription)?

Yes it does work (I am only using a single Twitter account, if that matters)

From my testing,
what works:
- Per profile Subscription
- Viewing Saved Posts
- Viewing **Posts** Opened from an external app (eg. https://twitter.com/kotatsu_muri_3/status/1724343459975434309 )

What doesn't:
- Feed (Blank)
- Groups (even with only 1 member it doesn't work, blank)
- Viewing **Accounts** Opened from an external app (eg https://twitter.com/kotatsu_muri_3 , doesn't load anything and will stay on previous tab)
- Search ("No results")
- Trending page (it shows worldwide trend, but clicking anything brings you to search and search is broken)

**Comment by BaconCondensate** [comment]
@j-fbriere Are there any lead to this issue yet?  It doesn't seem to happen in [Quacker](https://github.com/TheHCJ/Quacker).  But quacker was discontinued just today

**Comment by j-fbriere** [comment]
@BaconCondensate Just to be clear, the feed/groups/searches never worked for you with Squawker?

**Comment by BaconCondensate** [comment]
@j-fbriere After quite a bit more testing, I have found the exact issue for the bug (at least on mine), although I do not know how to exactly replicate the bug.

The issue arises from "tainted" configuration ( I would assume it was from older Squawker version?)
The fix is as follows:

1. Export Everything **Except** `Settings`
2. Clear Squawker app data
3. Re-import

**Comment by phourniner** [comment]
> @j-fbriere After quite a bit more testing, I have found the exact issue for the bug (at least on mine), although I do not know how to exactly replicate the bug.
> 
> The issue arises from "tainted" configuration ( I would assume it was from older Squawker version?) The fix is as follows:
> 
> 1. Export Everything **Except** `Settings`
> 2. Clear Squawker app data
> 3. Re-import

Sadly this did not work for me. :(
